ios-swiftui/realm-template-apps-swiftui/realm_template_apps_swiftuiApp.swift

```diff
@@ -0,0 +1,16 @@
+//let realmAppId = Bundle.main.object(forInfoDictionaryKey: "REALM_APP_ID") as? String
```

Comment:
What's this?

Reply:
Oops! That is leftover from when I was trying to hide the appID as a secret. Removing.